server/src/infra/ldap_handler.rs

```diff
@@ -538,6 +871,14 @@ impl<Backend: BackendHandler + LoginHandler + OpaqueHandler> LdapHandler<Backend
                     ]);
                 }
             }
+        } else if request.base == "cn=Subschema" && request.scope == LdapSearchScope::Base {
+            debug!("Subschema request made");
```

Comment:
Is there a separate schema request, or is that just the name for it? In which case, let's use "schema" in the debug message.